artifacthub/library/pod-security-policy/forbidden-sysctls/1.0.0/README.md

```diff
@@ -1,5 +1,36 @@
 # Forbidden Sysctls security context policy
 
-Forbidden sysctls excludes specific sysctls. You can forbid a combination of safe and unsafe sysctls in the list. To forbid setting any sysctls, use `*` on its own. If a sysctl pattern ends with a `*` character, such as `kernel.*`, it'll match `*` with rest of the sysctl.
+The forbidden sysctls constraint allows one to limit the set of kernel parameters that can be modified by pods. This is accomplished by specifying a combination of allowed and forbidden sysctls using either of two parameters: `allowedSysctls` and `forbiddenSysctls`.
```

Comment:
I believe 1.0.0 autogenerated changes should be removed now that the version has been bumped to 1.1.0 in the source.

Reply:
Reverted in 4fc2715c2ef1f0f1a58d302bcfb2cd97bdad1d2d